src/Microsoft.Identity.Web/Constants/IDWebErrorMessage.cs

```diff
@@ -42,6 +42,10 @@ internal static class IDWebErrorMessage
         public const string TenantIdClaimNotPresentInToken = "IDW10401: Neither `tid` nor `tenantId` claim is present in the token obtained from Microsoft identity platform. ";
         public const string ClientInfoReturnedFromServerIsNull = "IDW10402: Client info returned from the server is null. ";
         public const string TokenIsNotJwtToken = "IDW10403: Token is not JWT token. ";
+        public const string ClientCredentialScopeParameterShouldEndInDotDefault =
+       "IDW10404: 'scope' parameter should be of the form 'AppIdUri/.default'.";
+        public const string ClientCredentialTenantShouldBeTenanted =
```

Comment:
should an aka.ms link be included here w/more info on why this is the case? is it clear enough for developers to know how to fix it based on the error message? i'm not sure.

Reply:
Thanks for the suggestion. I've added one:  See https://aka.ms/ms-id-web/daemon-scenarios.